<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>851</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>851</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>480</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>发送文件</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/11.png</normaloff>images/11.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>发送图片</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/12.png</normaloff>images/12.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>31</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>设置</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/13.png</normaloff>images/13.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SendMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>526</x>
     <y>495</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>778</x>
     <y>495</y>
    </hint>
    <hint type="destinationlabel">
     <x>809</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SendFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SendGraphics()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>SetMessage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>80</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SendMessage()</slot>
  <slot>SendFile()</slot>
  <slot>SendGraphics()</slot>
  <slot>SetMessage()</slot>
 </slots>
</ui>
